root/report/instruments_without_image.tt

```diff
@@ -0,0 +1,42 @@
+[%- WRAPPER 'layout.tt' title=l('Instruments without image') full_width=1 -%]
```

Comment:
Grammar nitpick: Either "Instruments without an image" (each instrument has no image) or "Instruments without images" (none of them have images) would work. Same in the h1 and in the description below, but I would just make "relationships" plural there, since it's not known whether to use "a" or "an."